src/main/java/mate.academy/lib/Injector.java

```diff
@@ -1,13 +1,69 @@
 package mate.academy.lib;
 
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+import mate.academy.service.impl.ImplementationFinder;
+
 public class Injector {
     private static final Injector injector = new Injector();
+    private final Map<Class<?>, Object> instances = new HashMap<>();
```

Comment:
Consider making the `instances` map a class field and initializing it with `Map.of()` to improve readability and potentially make it immutable if no further modifications are expected.

Reply:
ok, but the instances, are created during the whole process in Injector class. How can I gathered them with simple initialization Map.of() ... ?